composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "ezsystems/ezpublish-kernel": "^6.7",
+        "ezsystems/ezpublish-kernel": "^6.7@dev",
```

Comment:
I'm not sure if this is the right place or the right way to require `dev-master` for `ezpublish-kernel` (took that from `ezplatform` repository), but I need this as translation removal is a part of unreleased `1.11` / `6.11`.